models/multiphase/d3q27_pf_velocity/model.R

```diff
@@ -0,0 +1,86 @@
+<?R
```

Comment:
This is a Rt file, but the suffix is ".R". If this is pure R, just get rid of `<?R` stuff.

Reply:
Nice catch, will do